src/main/resources/view/AppointmentListCard.fxml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+
+<HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
+  <GridPane HBox.hgrow="ALWAYS">
+    <columnConstraints>
+      <ColumnConstraints hgrow="SOMETIMES" minWidth="10" prefWidth="300" />
+    </columnConstraints>
+    <VBox alignment="CENTER_LEFT" minHeight="105" GridPane.columnIndex="0">
+      <padding>
+        <Insets bottom="5" left="15" right="5" top="5" />
+      </padding>
+      <HBox alignment="CENTER_LEFT" spacing="5">
+        <Label fx:id="id" styleClass="cell_big_label">
+          <minWidth>
+            <!-- Ensures that the label text is never truncated -->
+            <Region fx:constant="USE_PREF_SIZE" />
+          </minWidth>
+        </Label>
+        <Label fx:id="name" styleClass="cell_big_label" text="\$name" wrapText="true" />
+      </HBox>
+      <HBox alignment="CENTER_LEFT">
+        <padding>
+          <Insets top="6" />
+        </padding>
+        <VBox alignment="CENTER" minWidth="-Infinity" HBox.hgrow="NEVER">
+          <Label fx:id="startDate" styleClass="cell_big_light_label" text="\$start date" />
+          <Label fx:id="startTime" styleClass="cell_big_light_label" text="\$start time" />
+        </VBox>
+        <VBox alignment="CENTER_LEFT" minWidth="-Infinity" HBox.hgrow="NEVER">
+          <padding>
+            <Insets left="8" right="8" />
+          </padding>
+          <Label styleClass="cell_small_label" text="to" />
+        </VBox>
+        <VBox alignment="CENTER_LEFT" minWidth="-Infinity" HBox.hgrow="NEVER">
+          <Label fx:id="endDate" styleClass="cell_big_light_label" text="\$end date" />
+          <Label fx:id="endTime" styleClass="cell_big_light_label" text="\$end time" />
+        </VBox>
+        <AnchorPane>
+          <padding>
+            <Insets left="32" />
+          </padding>
+          <VBox alignment="CENTER_LEFT" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0">
+            <Label styleClass="cell_big_light_label" text="With" />
+            <Label fx:id="personName" styleClass="cell_small_label" text="\$Person Name" />
```

Comment:
minor nit since the above placeholder text is all lowercase, can consider changing this two to lower case also

Reply:
Sorry! You are right, fixed in commit 61c2e99.